### ISSUE TITLE
ENGL 402 - Week 13

### ISSUE THREAD

**Issue by taylorcate** [taylorcate/NuttingVariorum#30]
"Language Worlds" pg 25. Shipka  

"examining the communicative process _as a dynamic whole_" pg. 28 Shipka

"Specifically, DePew suggests researchers attend to the artifacts as well as to the composer's intentions, the audience's responses to the artifact, and to the local contexts shaping the artifact's production and reception" pg. 29 Shipka  

"Tracing the processes by which texts are produced, circulated, received, responded to, used, misused, and transformed, we are able to examine the complex interplay of the digital and analog, of the human and nonhuman, and of technologies, both new and not so new" pg. 30 Shipka  

"I am equally concerned with how _a narrow definition of technology_ fails to encourage richly nuanced, situated views of literacy_" pg. 31 Shipka  

"...an activity-based multimodal framework requires students to spend the semester attending to how language, _combined with still other representational systems,_ mediates communicative practice" (pp. 87-88 Shipka  

"_rhetorical sensitivity_" pg. 89 Shipkpa  

"Lost and Found" Assignment pg. 91 Shipka

**Comment by taylorcate** [comment]
[Shipkia Toward a Composition Made Whole Chs 1 and 4.pdf](https://github.com/taylorcate/NuttingVariorum/files/3062300/Shipkia.Toward.a.Composition.Made.Whole.Chs.1.and.4.pdf)

**Comment by taylorcate** [comment]
After reviewing the numerous pedagogies we've studied in this class, I can say without a doubt that multimodal approaches to composition make the most sense to me. As I was reading, I concocted multiple lessons and assignments in my mind that would work well in the multimodal classroom---some of which I plan to work into my syllabus for this class. Shipka's deep dive into the composition landscape and its connections to communication studies gave me a good understanding of how this movement grew in conjunction to the process movement and in response to more "conventional" or "traditional" composition pedagogies. I especially enjoyed and was taken with her emphasis on the mutability and infinite variety of technology both past and present. The multimodal individual is one we are already accustomed to seeing, loving, hating, talking to, and learning from. We are and have been for centuries beings who conventionalize the use of many types of technology in our everyday lives out of convenience, for survival, and to convey our innermost feelings. Why shouldn't this awareness of our past, present, and future dependency on "technology" (in all forms) make its way into the composition classroom? I think our students need to know what means of technology are available to them as tools to structure and dispense their arguments. 

I am especially interested in this idea of "_rhetorical sensitivity_" Shipka cites in Chapter Four of her book. Modern students are encountering rhetorical language in nearly every area of their lives, only heightened by their numerous and often unrestricted exposure to networked systems and social media. In short, our young people are taking in information from literally _any_ source with just as infinite intent and motive. If we structure out composition classes around teaching our students about rhetorical sensitivity, we could prepare them to think critically about where information is coming from and the potential rhetorical strategies its author employs to conjure a response from the reader. For example, the teacher could show a series of plain text quotes (all very stripped down, one quote per slide, black text on a white background) to the class and ask them to guess what work or medium they are from. The best example that comes to my mind now would be a tweet and then a line from a poem on memes. First you show the quote without the context and ask the students to make assumptions, then show them the text in relation to its medium to either refute or bolster those assumptions---this makes the student think critically about how the medium affects the perceived meaning of language. This is what I understood rhetorical sensitivity to be;  the ability to sense the rhetorical strategies present in many forms of communication. It is my goal to craft a class that really embraces this idea by asking students to analyze these various types of media but then to also craft similar types on their own. I have a feeling Dr. Hopwood will talk about her unessay assignment during class so I won’t say much about it here, but I believe an assignment like that and very similar to what Shipka describes in Chapter Four will soon populate many COMP syllabi in the coming years out of necessity to stimulate and engage such broadly connected students destined to live networked lives.
